### PR TITLE
feat: adjust colors

### LINKED ISSUE #393: Adjust colors
Adjust colors of chips, search bar button and in general align frontend with Figma colors.

### PR DESCRIPTION
**Pull request by janmichek**
## Description
fixes #393

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] adjust styles by visual material

### CODE REVIEW

File: src/components/AppChip.vue [hunk 1]
```diff
@@ -63,6 +63,11 @@ defineProps({
     color: var(--color-white);
   }
 
+  &--dark {
```
Comment (janmichek) [inline src/components/AppChip.vue:66]: following bootstrap naming convention

**Comment by janmichek** [comment]
Hope I didn't miss anything .) please help me check

File: src/components/AppHero.vue [hunk 2]
```diff
@@ -4,7 +4,7 @@
       <div class="hero__row">
         <div class="hero__column">
           <h1 class="hero__heading">
-            æternity {{ selectedNetwork }} at a glance
+            æternity at a glance
```
Comment (lukeromanowicz) [inline src/components/AppHero.vue:7]: Was it requested to change? If I remember correctly it was originally requested to be this way by Yani to give a user very visible information on which network is used.

**Review by lukeromanowicz — APPROVED** [review]
Looks really good to me but I don't have a good eye for details. I would involve Charis/Liubov here to check it out too.

File: src/pages/index.vue [hunk 3]
```diff
@@ -60,19 +60,24 @@
...
             being able to escalate on-chain in case of disagreement.
             <app-link
               class="dashboard__link"
-              to="https://aeternity.com/state-channels">
+              to="https://medium.com/aeternity-crypto-foundation/aeternity-bitcoin-ng-the-way-it-was-meant-to-be-df7bb1d65a4b"
+              is-text-link>
               Learn more
             </app-link>
+            <br>
+            Soon also with Hyperchains support!
```
Comment (michele-franchi) [inline src/pages/index.vue:76]: Probably a wrong copy paste?

File: src/pages/index.vue [hunk 4]
```diff
@@ -60,19 +60,24 @@
...
+        </div>
+        <div class="dashboard__column">
+          <p class="dashboard__paragraph dashboard__paragraph--horizontal">
             State Channels allow the gas-free execution of smart contracts and
             transactions, privately and with the speed of light, while still
             being able to escalate on-chain in case of disagreement.
             <app-link
               class="dashboard__link"
-              to="https://aeternity.com/state-channels">
+              to="https://medium.com/aeternity-crypto-foundation/aeternity-bitcoin-ng-the-way-it-was-meant-to-be-df7bb1d65a4b"
```
Comment (michele-franchi) [inline src/pages/index.vue:71]: The old one is correct, it's about state channels.

File: src/components/TheSearchBar.vue [hunk 5]
```diff
@@ -112,20 +121,28 @@ function isMicroblockId(query) {
...
+    color: var(--color-white);
   }
 
   &__input {
     width: 100%;
     border: none;
     background-color: var(--color-white);
     margin-right: var(--space-1);
-    font-size: 11px;
+    font-size: 14px;
```
Comment (michele-franchi) [inline src/components/TheSearchBar.vue:145]: I guess this is not needed anymore?

```
@media (--desktop) {
      font-size: 14px;
}
```

**Review by michele-franchi — CHANGES_REQUESTED** [review]
It looks good! I left some comments but in general I would change the following:

- State channels dashboard description looks kind of "broken" to me, I did like the previous version as I already commented on Figma
image

- For some reason the bold font in the navigation bar looks better in the design compared to the live version. I would revert it for now and retry again once we get the new fonts.

image

What do you think?

File: src/pages/index.vue (hunk 3 above)
Comment (janmichek) [reply src/pages/index.vue]: oops, reverted

File: src/pages/index.vue (hunk 4 above)
Comment (janmichek) [reply src/pages/index.vue]: reverted

File: src/components/TheSearchBar.vue (hunk 5 above)
Comment (janmichek) [reply src/components/TheSearchBar.vue:145]: RIght, not needed anymore. Removed

**Comment by janmichek** [comment]
> State channels dashboard description looks kind of "broken" to me, I did like the previous version as I already commented on Figma

No strong preference here. I reverted it

**Comment by janmichek** [comment]
> For some reason the bold font in the navigation bar looks better in the design compared to the live version. I would revert it for now and retry again once we get the new fonts.

It's a problem of rendering in Figma. I am also surprised that such a tool provides different results from the browser. Some other font will not resolve it

**Review by michele-franchi — CHANGES_REQUESTED** [review]
Thanks for the fixes, I noticed a few more things.
- I still think the weight font in the header does not fit well in this case
- Width of mobile header and width of the footer does not look aligned. I think there was a misunderstanding on Figma. You asked about the width but the reply was about the text in the footer:
> In short, easier readability. If i made it full-width gets too close to the links. I can make it all 50% of the full container if you want to have them identical or if its more practical to you :)

I don't think a different width is intentional and it's probably a leftover from the previous width, I would keep a single width. In my eyes things looks misaligned with different widths but that's my personal opinion.

**Comment by janmichek** [comment]
> Thanks for the fixes, I noticed a few more things.
> 
>     * I still think the weight font in the header does not fit well in this case
> 
>     * Width of mobile header and width of the footer are not aligned. I think there was a misunderstanding on Figma. You asked about the width but the reply was about the text in the footer:
> 
> 
> > In short, easier readability. If i made it full-width gets too close to the links. I can make it all 50% of the full container if you want to have them identical or if its more practical to you :)
> 
> I don't think a different width is intentional and it's probably a leftover from the previous width, I would keep a single width. In my eyes things looks misaligned with different widths but that's my personal opinion.

Ok, agree. I reverted it back

**Review by michele-franchi — APPROVED** [review]
It looks good now, thanks for fixing it.

Another thing I noticed is the header that is misaligned on mobile but feel free to handle it in a separate PR.
image

**Comment by janmichek** [comment]
I will merge this as it is hard for Liubov to test. It is vaguely defined and the visual material contains a lot of outdated things.

**Pull request merged**